src/functionalTest/java/uk/gov/hmcts/reform/bulkscan/orchestrator/AttachExceptionRecordToExistingCaseTest.java

```diff
@@ -130,6 +133,57 @@ public void should_attach_exception_record_to_the_existing_case_with_evidence_do
         verifyExistingCaseIsUpdatedWithExceptionRecordData(caseDetails, exceptionRecord, 1);
     }
 
+    @Test
+    public void should_attach_supplementary_evidence_with_pending_payments() throws Exception {
```

Comment:
Best to not necessarily call it `supplementary evidence`, but something more generic, because it's configuration-dependent. How about `should_attach_exception_record_with_pending_payments_when_classification_allows` or something like this?